### PR TITLE
fix(README.md and Eslint Error):Implement feedback - fix the typo in …

### PR DESCRIPTION
**Pull request by sulenchy**
…the README.md file - fix the eslint promise error [Fixes ##159261735]

### What does this PR do?
Implement feedback on  README.md and Eslint promise error

#### Description of Task to be completed?
- Update the README.md.
- Fix the eslint promise error.
#### How should this be manually tested?
N/A

#### Any background context you want to provide?
Feedback implementation

#### What are the relevant pivotal tracker stories?
Developer should implement the feedback

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import express from 'express';
 import logger from 'morgan';
 import bodyParser from 'body-parser';
 import dotenv from 'dotenv';
-import path from 'path';
+// import path from 'path';
```
Comment (fob413) [inline server/app.js]: Why do you have this commented out?

File: server/app.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ app.use(logger('dev'));
 
 const port = process.env.PORT || 3000;
 
-app.use(express.static(path.join(__dirname, '/public')));
+// app.use(express.static(path.join(__dirname, '/public')));
```
Comment (fob413) [inline server/app.js]: Why do you have this commented out?

**Review by fob413 — CHANGES_REQUESTED** [review]

File: server/app.js (hunk 2 above)
Comment (sulenchy) [reply server/app.js]: Eslint flaggeg my html static file. So I decided  to remove the static file and related line of code. I forgot to remove the commented line

Comment (fob413) [reply server/app.js]: Please go ahead and delete the commented code since it's not in use

Comment (sulenchy) [reply server/app.js]: Ok

**Review by fob413 — APPROVED** [review]

**Review by fob413 — APPROVED** [review]
Go ahead and merge

**Pull request merged**